docs/index.rst

```diff
@@ -3,6 +3,370 @@
+
+``logs.dir``
+  Top level HDFS directory to store the write ahead logs.
+
+  * Type: string
+  * Default: "logs"
+  * Importance: high
+
+``partitioner.class``
+  The partitioner to use when writing data to HDFS. You can use DefaultPartitioner, which preserves
+  the Kafka partitions; FieldPartitioner, which partitions the data to different directories
+  according to the value of the partitioning field specified in partition.field.name configuration;
+  TimebasedPartitioner, which partitions the data according to the time ingested to HDFS.
+
+  * Type: string
+  * Default: "io.confluent.connect.hdfs.partitioner.DefaultPartitioner"
+  * Importance: high
+
+``retry.backoff.ms``
+  The retry backoff in milliseconds. This config is used to notify Kafka connect to retry delivering
+  a message batch or performing recovery in case of transient exceptions.
+
+  * Type: long
+  * Default: 5000
+  * Importance: high
+
+``rotate.interval.ms``
+  The interval to rotate files.
+
+  * Type: long
+  * Default: -1
+  * Importance: high
+
+``schema.compatibility``
+  The schema compatibility rule to use observing changes during data
+
+  * Type: string
+  * Default: "NONE"
+  * Importance: high
+
+``topics.dir``
+  Top level HDFS directory to store the data ingested from Kafka.
+
+  * Type: string
+  * Default: "topics"
+  * Importance: high
+
+``locale``
+  The locale to use when partitioning with TimeBasedPartitioner.
+
+  * Type: string
+  * Default: ""
+  * Importance: medium
+
+``partition.duration.ms``
+  The duration of a partition milliseconds used by TimeBasedPartitioner.
+
+  * Type: long
+  * Default: -1
+  * Importance: medium
+
+``partition.field.name``
+  The name of the partitioning field when FieldPartitioner is used.
+
+  * Type: string
+  * Default: ""
+  * Importance: medium
+
+``path.format``
+  The format of the path to use when partitioning with TimeBasedPartitioner.
+
+  * Type: string
+  * Default: ""
+  * Importance: medium
+
+``schema.cache.size``
+  The size of the schema cache used in the Avro converter.
+
+  * Type: int
+  * Default: 1000
+  * Importance: medium
+
+``shutdown.timeout.ms``
+  Clean shutdown timeout. This makes sure that asynchronous Hive metastore updatesare completed
+  during connector shutdown.
+
+  * Type: long
+  * Default: 3000
+  * Importance: medium
+
+``storage.class``
+  The storage layer.
+
+  * Type: string
+  * Default: "io.confluent.connect.hdfs.storage.HdfsStorage"
+  * Importance: medium
+
+``timezone``
+  The timezone to use when partitioning with TimeBasedPartitioner.
+
+  * Type: string
+  * Default: ""
+  * Importance: medium
+
+``filename.offset.zero.pad.width``
+  Width to zero pad offsets in HDFS filenames to if the offsets is too short in order to provide
+  fixed width filenames that can be ordered by simple lexicographic sorting.
+
+  * Type: int
+  * Default: 10
+  * Importance: low
+
+``hive.database``
+  The database to use when the connector create tables in Hive.
```

Comment:
Should probably be "high" priority. Every reasonable customer will want to change this...